### PR TITLE
Replace robocoup mk. 1 with robocoup mk. 2. Closes #129.

### LINKED ISSUE #129: complete migration of robocoup to chatter
primary work to be done here:
https://github.com/bocoup/bots/blob/master/src/new-bot/commands/expertise.js

### PR DESCRIPTION
**Pull request by cowboy**

### CODE REVIEW

**Comment by cowboy** [comment]
@tkellen review plz thx u

**Comment by tkellen** [comment]
doing so now!

**Comment by tkellen** [comment]
That is a wonderful amount of deletions 💃

**Comment by cowboy** [comment]
> That is a wonderful amount of deletions 💃

I deleted the only test file in the repo tho. However, chatter has a ton of tests, so it's a net testing win!

File: src/robocoup/commands/utilization.js [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
+  description: 'Show information about our utilization rate.',
+}, () => {
+  return query('utilization_metric').get(0).then(metric => {
+    return [
+      `> *Utilization Status as of ${moment().format('MMMM Do, YYYY')}*`,
+      `> During the *last 30 days* our utilization rate was *${metric.last_30_days}%*.`,
+      `> During the *next 30 days* our utilization rate is currently scheduled to be *${metric.next_30_days}%*.`,
+      `> Year to date, our utilization rate has been *${metric.ytd}%*.`,
+      heredoc.oneline.trim`
+        > Our utilization goal is *80%* of our *${metric.billable_count}* billable team members
```
Comment (tkellen) [inline src/robocoup/commands/utilization.js]: is this for supporting 80 columns max?

Comment (tkellen) [inline src/robocoup/commands/utilization.js]: if yes, n/p, but in general i don't like the idea of adding runtime complexity for code style conformance

**Comment by tkellen** [comment]
:+1:

**Pull request merged**